src/debuginfo.cpp

```diff
@@ -803,12 +836,13 @@ static objfileentry_t &find_object_file(uint64_t fbase, StringRef fname) JL_NOTS
     (void)iswindows;
 
 // GOAL: Read debuginfo from file
-    // TODO: need read/write lock here for objfilemap synchronization
-    obfiletype::iterator it = objfilemap.find(fbase);
-    if (it != objfilemap.end())
-        // Return cached value
-        return it->second;
-    auto &entry = objfilemap[fbase]; // default initialized
+    objfileentry_t entry{nullptr, nullptr, 0};
+    {
```

Comment:
what is the random scope block for? temporaries are usually destroyed by `;`

Reply:
It's an artifact of when I was doing a find-and-set manually, rather than just directly using emplace. I'll get rid of this one plus the lower one too.